pinephone/device.mk

```diff
@@ -42,7 +42,7 @@ PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/modem.pinephone.rc:$(TARGET_COPY_OUT_VENDOR)/etc/init/modem.pinephone.rc \
 
 PRODUCT_COPY_FILES += \
-    device/glodroid/pinephone/audio.pinephone.xml:$(TARGET_COPY_OUT_VENDOR)/etc/audio.pinephone.xml \
+    device/glodroid/pinephone/audio/audio.pinephone.xml:$(TARGET_COPY_OUT_VENDOR)/etc/audio.pinephone.xml \
```

Comment:
Hello. Thanks for this series. I'm just wondering what is the rationale for this change?

Reply:
Maybe some 'audio_policy_configuration' will be added in the future, so it is better to keep it in a separate directory.

Reply:
Oh, i see, thanks. Please drop this patch from these series and apply it to series with `audio_policy_configuration` when ready.

Reply:
Done